docs/07-tools.md

```diff
@@ -246,3 +246,22 @@ gulp.task('build', async function () {
   });
 });
 ```
+
+### Deno
+If you like to run Rollup in you can use [esm.sh](https://esm.sh/) like so:
```

Comment:
the "in" seems to be a bit lost. did you mean "with Deno" maybe?

Reply:
Should be fixed now!